llvm/lib/Transforms/Scalar/MemCpyOptimizer.cpp

```diff
@@ -1524,8 +1525,8 @@ bool MemCpyOptPass::performStackMoveOptzn(Instruction *Load, Instruction *Store,
   };
 
   auto CaptureTrackingWithModRef =
-      [&](Instruction *AI,
-          function_ref<bool(Instruction *)> ModRefCallback) -> bool {
+      [&](Instruction *AI, function_ref<bool(Instruction *)> ModRefCallback,
+          SmallSet<Instruction *, 4> &AllocaInstUsersWithTBAA) -> bool {
```

Comment:
No need for the argument, the variable is captured.

Reply:
Done